docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -35,38 +37,54 @@ abstract class Client {
     account: string,
   }): Promise<Record<string, NotifySubscription>>;
 
-  // get all messages for a subscription
-  public abstract getMessageHistory(params: {
+  // get all notifications for a subscription
+  public abstract getNotificationHistory(params: {
     topic: string,
-  }): Promise<Record<number, NotifyMessageRecord>>
+  }): Promise<Record<number, NotifyNotificationRecord>>
 
   // delete active subscription
   public abstract deleteSubscription(params: {
     topic: string,
   }): Promise<void>;
   
-  // delete notify message
-  public abstract deleteNotifyMessage(params: {
+  // delete notification
+  public abstract deleteNotification(params: {
     id: number,
   }): Promise<void>;
   
   // decrypt notify subscription message
-  public abstract decryptMessage(params: {
+  public abstract decryptNotification(params: {
```

Comment:
it should not be a client method
could we do it like we do in w3w there the method is a part of an interface and depending on implementation client can optionally conform to this interface?

Reply:
Can you link to how we do it in w3w? thoughts @Elyniss @chris13524 @flypaper0  ?